horovod/run/run.py

```diff
@@ -454,25 +466,40 @@ def run():
             'training script using the standard way provided by your'
             ' MPI distribution (usually mpirun, srun, or jsrun).')
 
+    # Pass all the env variables to the mpirun command.
```

Comment:
Can we also pack all the MPI run code (essentially rest of this file) into a mpi_run file, similar to gloo_run?